Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -574,6 +574,36 @@ static RefPtr<CSSValue> positionOffsetValue(const RenderStyle& style, CSSPropert
     return CSSPrimitiveValue::create(CSSValueAuto);
 }
 
+// FIXME: Replace calls to this function with `getCSSPropertyValuesForShorthandProperties`
+// once white-space longhands are completely implemented.
```

Comment:
I don't think we can completely replace this with `getCSSPropertyValuesForShorthandProperties` fwiw, we still want to provide the shortest serialization, which `getCSSPropertyValuesForShorthandProperties` won't necessarily provide.

I'd just remove this FIXME.